CreeDictionary/tests/API_tests/model_test.py

```diff
@@ -194,3 +195,44 @@ def deep_contain(container: Iterable, testee):
     assert deep_contain(nipaw_paradigm, "kinipân")
     assert deep_contain(nipaw_paradigm, "nipâw")
     assert deep_contain(nipaw_paradigm, "ninipânân")
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("query", ["nipaw", "nitawi", "nitawi-nipaw", "e-nitawi-nipaw"])
+def test_search_serialization_json_parsable(query):
+    """
+    Test SearchResult.serialize produces json compatible results
+    """
+    results = Wordform.search(query)
+    for result in results:
+
+        serialized = result.serialize()
+        try:
+            json.dumps(serialized)
+        except Exception as e:
+            print(e)
+            pytest.fail("SearchResult.serialized method failed to be json compatible")
+
+
+@pytest.mark.django_db
+def test_search_words_with_preverbs():
+    """
+    preverbs should be extracted and present in SearchResult instances
+    """
+    results = Wordform.search("nitawi-nipâw")
+    assert len(results) == 1
+    search_result = results.pop()
+
+    assert len(search_result.preverbs) == 1
+    assert search_result.preverbs[0].text == "nitawi-"
+
+
+@pytest.mark.django_db
+def test_search_text_with_ambiguous_word_classes():
```

Comment:
I initially used the name `test_search_text_with_ambiguous_pos()`, but then I remembered in the glossary, pos has been superceded by "word class".

This should be a good start of using the new glossary for me